pkg/karmadactl/cmdinit/cmdinit.go

```diff
@@ -125,6 +125,8 @@ func NewCmdInit(parentCommand string) *cobra.Command {
 	flags.StringVar(&opts.ExternalIP, "cert-external-ip", "", "the external IP of Karmada certificate (e.g 192.168.1.2,172.16.1.2)")
 	flags.StringVar(&opts.ExternalDNS, "cert-external-dns", "", "the external DNS of Karmada certificate (e.g localhost,localhost.com)")
 	flags.DurationVar(&opts.CertValidity, "cert-validity-period", cert.Duration365d, "the validity period of Karmada certificate (e.g 8760h0m0s, that is 365 days)")
+	flags.StringVarP(&opts.CaCertPath, "ca-cert-path", "", "", "The path of CA cert, if not set, will gen new, the option must be specified together with ca-key-path")
+	flags.StringVarP(&opts.CaKeyPath, "ca-key-path", "", "", "The path of CA key, if not set, will gen new,  the option must be specified together with ca-cert-path")
```

Comment:
I would prefer to `file` than `path` as it is more straightforward. Also suggesting usages:
```suggestion
	flags.StringVarP(&opts.CaCertPath, "ca-cert-file", "", "", "The root CA certificate file which will be used to issue new certificates for Karmada components. If not set, a new self-signed root CA certificate will be generated. This must be used together with --ca-key-file.")
	flags.StringVarP(&opts.CaKeyPath, "ca-key-file", "", "", "The root CA private key file which will be used to issue new certificates for Karmada components. If not set, a new self-signed root CA key will be generated. This must be used together with --ca-cert-file.")
```

Reply:
By the way, given the two flags must be used together, we might need to add validation for that, probably at :
https://github.com/karmada-io/karmada/blob/09f960085395a4f45ea49e23ee981afca3a2ce7b/pkg/karmadactl/cmdinit/kubernetes/deploy.go#L224-L242

Reply:
ok